src/bootstrap/native.rs

```diff
@@ -38,6 +38,7 @@ use cache::Interned;
 pub struct Llvm {
     pub target: Interned<String>,
     pub emscripten: bool,
+    pub dump_enabled: bool,
```

Comment:
There should be no need for this here as AFAICT it's always globally set, we never do anything but pass it in from config.llvm_dump_enabled.